src/components/StaticApp/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC, memo } from 'react';
```

Comment:
Do we actually need to duplicate the `ActionButton` in `static` app or can we just use `component/ActionButton`.  It doesn't look different at a glance.

Reply:
Short answer: no.
Long answer: I duplicated all of the components just because a) I didn't want to take the time to figure out which files could stay and which files had to change and b) I don't know how long this temporary fix will be live for, and I didn't want to cause possibly unforseen issues on the github demo because the static app was referencing a file which changed to be non-static. 

I can certainly go through and update these if its worth our time but for all future PRs until the useLocalStorage task is merged, we will need to be extra vigilant about whether or not the changes break the demo

Reply:
Updated to only include required components